modules/weave-scope/variables.tf

```diff
@@ -0,0 +1,8 @@
+variable "namespace" {
+  type    = string
+  default = "default"
```

Comment:
I think this should not go into default namespace. meta-system, dasmeta-system?

Reply:
gotcha